go/test/endtoend/cluster/mysqlctl_process.go

```diff
@@ -249,6 +257,22 @@ func MysqlCtlProcessInstanceOptionalInit(tabletUID int, mySQLPort int, tmpDirect
 	return mysqlctl
 }
 
+func isSQLFlavor() (bool, error) {
```

Comment:
We are only using this to check if it supports super-read-only to find which init_db.sql to use, right? Maybe we can have a utility function like `GetInitDBSQLFilename()` which has the logic in the file and returns the correct init filename?